TCHAP_CHANGES.md

```diff
@@ -0,0 +1,37 @@
+Changes in Tchap 2.0.0 (2021-xx-xx)
+===================================================
+
+First release!
+
+
+=======================================================
++        TEMPLATE WHEN PREPARING A NEW RELEASE        +
+=======================================================
```

Comment:
Please move the template at the top, if you agree

Reply:
What do you mean by at the top ?

Reply:
Ignore this remark, I misunderstood the meaning of this template